### ISSUE TITLE
[Feature Request] Translate from Imperial/US system of measurement to Metric system.

### ISSUE THREAD

**Issue by ris58h** [mdnsk/leo-translate#17]
Examples: 
12.5'' should be translated to 31,75см
5lbs to 2,27кг
and so on.

I would like to implement this feature. Don't you mind?

**Comment by mdnsk** [comment]
It would be great!
But, for the upcoming release, this feature should be disabled by default (on the option page) to save stability of the add-on.

**Comment by ris58h** [comment]
I did some research and I'm not sure about this feature now.
Leo-translate already do translate things like "4.2lbs" to "4.2фунтов", but do not convert them to the metric system.
I found [autoConverter](https://chrome.google.com/webstore/detail/autoconvert-auto-currency/pbghjbbdfkjffaalmcfgelakmeckapnd) (Chrome only) web-extension that convert values between measurement systems. May be it would be useful to make separate extension with such logic.
